ml_pipeline_engine/node/node.py

```diff
@@ -134,7 +126,7 @@ def build_node(
     run_method = get_run_method(node)
     if not run_method:
         raise RunMethodExpectedError(
-            f'Ожидается наличие хотя бы одного run-метода. methods={NodeBase.RUN_METHOD_ALIASES}',
+            f'Missing method for node execution. Expected name={NodeBase.RUN_METHOD_ALIAS}',
```

Comment:
Also we should check it in main validation in the builder

Reply:
I found an interesting side effect;

currently it is possible to build a graph with nodes without "process"  method; 
but if we change the following statement ([/ml_pipeline_engine/dag_builders/annotation/builder.py#L61](https://github.com/tochka-public/ml-pipeline-engine/blob/breaking_change/remove_nodelike/ml_pipeline_engine/dag_builders/annotation/builder.py#L61))

```python
        if not annotations and parameters:
            raise errors.UndefinedAnnotation(f'Невозможно найти аннотации типов. obj={obj}')
```
to the statement that checks each argument
```python
        if not annotations and not parameters:
            raise errors.UndefinedAnnotation(f'Невозможно найти аннотации типов. obj={obj}')
```
then building a graph without "process" is going to raise the Exception

will it be enough to resolve the current tread ?

Reply:
I've added a raise of exception to the function `get_callable_run_method` instead of just returning a None object
4d32382edc56bab14a3b61af2649e379eef2c601